pkg/scanfetcher/nexpose_test.go

```diff
@@ -189,6 +190,7 @@ func TestNexposeClient_FetchScans(t *testing.T) {
 			nexposeClient := &NexposeClient{
 				Client:   &http.Client{Transport: mockRT},
 				Endpoint: endpoint,
+				StatFn:   runhttp.StatFromContext,
```

Comment:
task: this needs to use a stat client stub/mock instead of the actual stat function.

Reply:
no longer necessary due to comment above